packages/ics721/src/execute.rs

```diff
@@ -325,12 +344,18 @@ where
     }
 
     /// Default implementation using `cw721_base::msg::InstantiateMsg`
-    fn init_msg(&self, _deps: Deps, env: &Env, class: &Class) -> StdResult<Binary> {
+    fn init_msg(&self, deps: Deps, env: &Env, class: &Class) -> StdResult<Binary> {
+        // use ics721 creator for withdraw address
+        let ContractInfoResponse { creator, .. } = deps
+            .querier
+            .query_wasm_contract_info(env.contract.address.to_string())?;
+
         // use by default ClassId, in case there's no class data with name and symbol
         let mut instantiate_msg = cw721_base::msg::InstantiateMsg {
             name: class.id.clone().into(),
             symbol: class.id.clone().into(),
             minter: env.contract.address.to_string(),
+            withdraw_address: Some(creator),
```

Comment:
Allow creator of ics721 for setting withdraw address on nft contract (allowing to send and withdraw funds). This affected only ics721-base for permissionless chains.
sg-ics721 is unchanged.
More on withdraw address, see here: https://github.com/CosmWasm/cw-nfts/pull/149